nixos-generate

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 
 ## Configuration
 
-libexec_dir=$(dirname "$0")
+libexec_dir="${0%/*}"
```

Comment:
wuhuu, nice trick. Thats the kind of magic I like ;D

Reply:
hehe :)